packages/trpc-playground/src/handlers/common.ts

```diff
@@ -18,12 +18,12 @@ export const getCommonHandlerReqData = async (config: TrpcPlaygroundConfig) => {
     })
   }
 
-  const types = await resolvedConfig.resolveTypes(config.router)
+  const types = await resolvedConfig.resolveTypes!(config.router)
 
   return {
     config: resolvedConfig,
     htmlPlaygroundPage,
-    stringifiedTypes: JSON.stringify(types),
+    types,
```

Comment:
Why? These are stringified here because they don't change, so we only need to call `JSON.stringify` once, not per-request

Reply:
Yes, that makes sense.
fixed